File: lab05/echon.pl
#!/usr/bin/perl -w
if (@ARGV!=2) {
    print  "Usage: $0 <number of lines> <string> at ./$0 line 3.\n";
    exit
}

for ($i = 0; $i < $ARGV[0]; $i++) {
    print "$ARGV[1]\n";
}

File: lab06/echon.py
#!/usr/bin/python
import sys
if len(sys.argv) != 3:
    print('Usage: ./echon.py <number of lines> <string>')
    exit()
try:
    for i in range(int(sys.argv[1])):
        print(sys.argv[2])
except ValueError:
    print('./echon.py: argument 1 must be a non-negative integer')
    exit()
